jep/10/README.adoc

```diff
@@ -0,0 +1,142 @@
+= JEP-10: Jenkins Twitter Policy for Contributors
+:toc: preamble
+:toclevels: 3
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+.Metadata
+[cols="1h,1"]
+|===
+| JEP
+| 10
+
+| Title
+| Jenkins Twitter Policy for Contributors
+
+| Sponsor
+| link:https://github.com/tracymiranda[Tracy Miranda]
+
+| Status
+| Draft :speech_balloon:
+
+| Type
+| Process
+
+| Created
+| 2018-08-07
+
+| BDFL-Delegate
+| link:https://github.com/rtyler[R. Tyler Croy]
+
+|===
+
+== Abstract
+
+This JEP describes the policy for Jenkins twitter accounts and covers:
+
+* Enabling a team of contributors
+* Jenkins Twitter policy
+* Regular reports on Twitter engagement
+
+== Specification
+
+=== Enabling a Team of Contributors
+
+Tweetdeck's Team feature will be used to manage the team of contributors.
+The following roles are recognized:
+
+* Owner - owner of the twitter account
+* Admin - can invite others to contribute
+* Contributor - creates content and can tweet, retweet and schedule tweets
+
+New contributors may be proposed at a governance meeting.
```

Comment:
Everything at the governance meeting should be actually proposed in the mailing lists first. I would rather require an email to the Dev list

Reply:
Good point - I'll update this to reflect that.